src/lib/common/statistics.cpp

```diff
@@ -192,7 +193,7 @@ void statisticsUpdate(RequestType request, Format inFormat)
   case RtUnsubscribeContextResponse:                     ++noOfRtUnsubscribeContextResponse; break;
   case RtSubscribeResponse:                              ++noOfRtSubscribeResponse; break;
   case RtSubscribeError:                                 ++noOfRtSubscribeError; break;
-
+  case RtContextElementResponse:                         ++noOfContextElementResponse; break;
```

Comment:
So one counter was missing until now and this is the bugfix? Need to mention in CHANGES_NEXT_RELEASE or in this case is a very tiny bugfix?